### PR TITLE
BZ1886291: Added ports and wildcard URL with note

### PR DESCRIPTION
**Pull request by skrthomas**
Direct link to doc preview: https://deploy-preview-31617--osdocs.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall.html

Added ports and wildcard URL with a note about allowing the wildcard.

https://bugzilla.redhat.com/show_bug.cgi?id=1886291

Verifying versions with SMEs.

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -13,103 +13,133 @@ There are no special configuration considerations for services running on only c
...
 
 |`openshift.org`
+|443, 80
 |Provides {op-system-first} images
+
 |===
 +
 When you add a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, then image downloads are denied when the initial download request is redirected to a host name such as `cdn01.quay.io`.
++
+CDN host names, such as `cdn01.quay.io` are covered when you add a wildcard entry such as `*.quay.io` in your allowlist.
```
Comment (ahardin-rh) [inline modules/configuring-firewall.adoc:44]: names, such as `cdn01.quay.io`, are
entry, such as `*.quay.io`, in

**Comment by ahardin-rh** [comment]
Just some minor suggestions from me!

**Comment by gpei** [comment]
LGTM

**Pull request merged**